mlrun/projects/project.py

```diff
@@ -3797,6 +3798,62 @@ def delete_api_gateway(
 
         mlrun.db.get_run_db().delete_api_gateway(name=name, project=self.name)
 
+    def store_alert_config(self, alert_data: AlertConfig, alert_name=None):
+        """
+        Create/modify an alert.
+        :param alert_data: the data of the alert.
+        :param alert_name: name of the alert.
+        """
+        db = mlrun.db.get_run_db(secrets=self._secrets)
+        if alert_name is None:
+            alert_name = alert_data.name
+        return db.store_alert_config(alert_name, alert_data.dict(), self.metadata.name)
+
+    def get_alert_config(self, alert_name):
+        """
+        Retrieve an alert.
+        :param alert_name: name of the alert to retrieve.
+        """
+        db = mlrun.db.get_run_db(secrets=self._secrets)
+        return db.get_alert_config(alert_name, self.metadata.name)
+
+    def list_alerts_configs(self):
+        """
+        Retrieve list of alerts of a project.
+        """
+        db = mlrun.db.get_run_db(secrets=self._secrets)
+        return db.list_alerts_configs(self.metadata.name)
+
+    def delete_alert_config(self, alert_data: AlertConfig = None, alert_name=None):
+        """
+        Delete an alert.
+        :param alert_data: the data of the alert.
+        :param alert_name: name of the alert to delete.
+        """
+        if alert_data is None and alert_name is None:
+            raise ValueError(
+                "At least one of alert_data or alert_name must be provided"
+            )
+        db = mlrun.db.get_run_db(secrets=self._secrets)
+        if alert_data:
+            alert_name = alert_data.name
+        return db.delete_alert_config(alert_name, self.metadata.name)
+
+    def reset_alert_config(self, alert_data=None, alert_name=None):
+        """
+        Reset an alert.
+        :param alert_data: the data of the alert.
```

Comment:
:param alert_data: -- The data of the alert.
        :param alert_name: -- The name of the alert to reset.